pkg/loki/loki.go

```diff
@@ -261,7 +261,7 @@ func New(cfg Config) (*Loki, error) {
 		Cfg:           cfg,
 		clientMetrics: chunk_storage.NewClientMetrics(),
 	}
-
+	usagestats.Edition("oss")
```

Comment:
nit: I feel this would be better in `RegisterFlags`

Reply:
I plan to overrides this in enterprise and I need a way to know when it happens hence why I move it inside the New here. So I can override after New.